handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -166,11 +265,11 @@ func getClaim(t *jwt.Token, claim string) (interface{}, bool) {
 	if claims, ok := t.Claims.(jwt.MapClaims); ok {
 		c, has := claims[claim]
 		if !has {
-			log.Printf("Missing claim %q for token %v", claim, t.Raw)
+			log.Printf("Missing claim %q for token", claim)
```

Comment:
Just log a short prefix or suffix or even both. That should be enough to troubleshoot against ongoing issues around the same time.